docs/_documentations/mdt-eclipse-importedprojects.md

```diff
@@ -23,6 +23,12 @@ Codewind is designed to develop cloud native microservices. Therefore, each proj
 * [Node.js projects](#nodejs-projects)
 * [Swift projects](#swift-projects)
 * [Generic Docker projects](#generic-docker-projects)
+* [Binding Appsody projects](#binding-appsody-projects) 
```

Comment:
I think it's fine to leave out the word "Binding" here

Reply:
Good call. I've made the change. Thanks.